### PR TITLE
Updating Docs

### PR DESCRIPTION
**Pull request by nileshprasad137**
I tried to update Documentation by adding some beginner friendly instructions !

### CODE REVIEW

**Comment by teryhill** [comment]
@nileshprasad137  here is a list of the settings and minimum values I make to the php.ini for a new install also make sure the time zone is set correctly.

Perhaps an expanded document in the /documents directory would be useful.

- [ ] short_open_tag = On
- [ ] max_execution_time = 600
- [ ] max_input_time = 600
- [ ] max_input_vars = 5000
- [ ] memory_limit = 1024M
- [ ] error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT & ~E_NOTICE
- [ ] register_argc_argv = On
- [ ] post_max_size = 32M
- [ ] upload_max_filesize = 16M
- [ ] session.gc_maxlifetime = 14400

**Comment by nileshprasad137** [comment]
okay, i'll make the necessary changes!

**Comment by nileshprasad137** [comment]
@teryhill Have a look now . I've covered windows installation part.

**Comment by teryhill** [comment]
@nileshprasad137  Looks good

**Comment by aethelwulffe** [comment]
We now have two pull requests on this.  Did you guys assign yourself to the issue?  Let's review both pull requests and try to attend to the overlap.  I am seeing that at this point, collaboration between the two authors might be a real good thing.
  This piece is one of the most important issues we have at this moment.   @azwreith #449

**Comment by nileshprasad137** [comment]
@aethelwulffe okay , so @azwreith has worked on install.md and I have worked on readme.md. In my opinion , i could modify readme.md a bit and then I could point out that , to have a detailed installation instruction, please refer to install.md. I'll also add installation instruction for linux by referring that install.md.

would that be fine?

**Comment by aethelwulffe** [comment]
That sounds the best idea to me.
If you feel like it is best to separate documentation by OS, and you need to add files making the readme more like an index, then please do so.
  The big question is: Are you guys are willing to revamp whole documents (looking at all the text and checking the links etc...) or if you would rather just get these commits in there so the server settings get posted ASAP is more important.  I think either approach is fine.  I just wanted to point out that there is still a lot of bad and outdated info in these docs.  If it is later, then we will need to keep an issue open.

File: README.md [hunk 1]
```diff
@@ -22,3 +22,55 @@ Tax deductible Donations can be made to the project through the SFC.
...
+* `register_argc_argv` = On
+* `post_max_size` = 32M
+* `upload_max_filesize` = 16M
+* `session.gc_maxlifetime` = 14400
+
+Make sure you have disabled strict mode in Mysql . 
+
+## How to disable Mysql strict mode?
+
+Make the following changes in the "my.ini/my.cnf":
```
Comment (aethelwulffe) [inline README.md]: Sometimes good to add the full package path for xampp, wamp, linux, MariaDB etc..

Comment (nileshprasad137) [inline README.md]: okay, i'll add the necessary changes!

File: README.md [hunk 2]
```diff
@@ -22,3 +22,55 @@ Tax deductible Donations can be made to the project through the SFC.
...
+
+    1.  Look for the following line:
+        sql-mode = "STRICT_TRANS_TABLES,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
```
Comment (aethelwulffe) [inline README.md]: Is it sql-mode in all cases, or sometimes sql_mode?

**Comment by aethelwulffe** [comment]
Please bro,
clean up anything you feel you have energy for.  This file was seriously outdated.

File: README.md (hunk 2 above)
Comment (nileshprasad137) [inline README.md]: In my case , its sql-mode. Not sure if it is the same everywhere. It should be same

**Comment by nileshprasad137** [comment]
I'll make all the neccesary changes tomorrow after i return back from classes.

File: README.md (hunk 2 above)
Comment (aethelwulffe) [inline README.md]: I really hope it is...but I have been proven horribly wrong in the past.  
Looks like that might be the case though.
https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html
I think sql-mode is the option file version, and the CLI/runtime option is sql_mode...I think...

**Comment by azwreith** [comment]
@aethelwulffe I did check the links and fix some of the text while translating it into markdown. The one thing I noticed during installation was that `libreehr/custom/letter_templates` was not present anymore in the files and should be removed (at least, it wasn't present in my files. Need someone to confirm).

**Comment by nileshprasad137** [comment]
I have some important academic work, I'll take care of this in some time.

**Comment by aethelwulffe** [comment]
letter_templates does indeed seem to be gone.

**Comment by aethelwulffe** [comment]
This looks ready to go man.  I have another commit out there renovating the actual /documentation directory, but this has nothing to do with that.  Good work, looks clean.
Is WAMPP only available on SourceForge?  I do tend to steer things away from SourceForge these days due to some liberties they have taken with .exe contents and installing adware.

**Comment by nileshprasad137** [comment]
@aethelwulffe I just updated Wamp's link. The only thing remaining according to me is to link INSTALL.MD

**Comment by nileshprasad137** [comment]
@aethelwulffe Currently the install.md file isn't in the main branch , so I can add that later when @azwreith is done. 
Would that be fine?

**Comment by aethelwulffe** [comment]
I like it.

**Comment by aethelwulffe** [comment]
Check out #454 as well.  You can see where I am going there.  That is a reorganization of the /documentation as well as adding some stuff for the windows install helper.

**Comment by teryhill** [comment]
@tmccormi  this has 2 up votes

**Comment by nileshprasad137** [comment]
Just for reference . Issues addressed through last commit : #438 #338

**Comment by aethelwulffe** [comment]
Terry would like this merged as well, so I am going ahead with it despite the commit history being...extensive :)

**Pull request merged**